### ISSUE TITLE
Meshnodes can't be tools / Non-tools can't have indicator bar on hotbar display.

### ISSUE THREAD

**Issue by jastevenson303** [minetest/minetest#6448]
![screenshot_20170920_052425a](https://user-images.githubusercontent.com/23448162/30636784-699318cc-9dc4-11e7-913b-a63eade13a13.png)

Plus is would be better if the wielditem could be translucent.

> Non-tools can't have wear
> Tools can't be meshnodes

So you can never hold a meshnode tool, like in the picture, with item_wear indicator bar on hotbar display.

**Comment by paramat** [comment]
Please can you split these up into 3 separate requests? However there's already a request issue somewhere for mesh wielditems, so 2 requests.

**Comment by tobyplowy** [comment]
> However there's already a request issue somewhere for mesh wielditems

I think that issue was made by me but I can't find it anywhere :/
I will keep looking tho
Also I think it's time to add mesh support for items (proper support no workarounds) its way overdo!

**Comment by paramat** [comment]
Actually the request for mesh wield to support translucency can be added to the existing issue, so this issue can be for 'Non-tools can't have indicator bar on hotbar display'.

**Comment by paramat** [comment]
#4190

**Comment by paramat** [comment]
I don't understand why you would need a wear indicator for a non-tool?

**Comment by tobyplowy** [comment]
@paramat yes that's the one i guess you beat me well better luck next time :)

> I don't understand why you would need a wear indicator for a non-tool?

Maybe to make it do something it Looks Like a magic Crystal To Me So Maybe for teleporting or something

**Comment by paramat** [comment]
PR #4715

**Comment by paramat** [comment]
Any core dev support? (i'm neutral).

**Comment by paramat** [comment]
@SmallJoker since you reviewed the PR #4715 do you support this request?

**Comment by jastevenson303** [comment]
It is indeed two issues in one, but I had to pick.

**Comment by rubenwardy** [comment]
I support this. This is also essentially a duplicate of the issue to combine tools and craftitems

**Comment by SmallJoker** [comment]
@paramat Yes I do. I'd even welcome to replace craftitems with tools overall. They're already very similar in functionality.

**Comment by rubenwardy** [comment]
Closed PR: #4715

Issue on merging tools and items: #4815

I feel like the best fix would be to
